src/main/java/co/omise/requests/Requester.java

```diff
@@ -15,34 +15,35 @@
  */
 public interface Requester {
     /**
-     * Sends the user generated {@link Request<T>} to {@link Requester} for it to be carried out
+     * Sends the user generated {@link Request} to {@link Requester} for it to be carried out
      *
      * @param <T>     the {@link Model} object type that is expected to be returned
      * @param <R>     the {@link Request} object type that is passed in from the user
-     * @param request the {@link Request<T>} user generated request
+     * @param request the {@link Request} user generated request
      * @param klass   the type of the object that the response is expected to be deserialized as
-     * @return the {@link Model} object of type <T>
+     * @return the {@link Model} object extending from Model
```

Comment:
I think we can have a better description of what object that would be returned from these methods

Reply:
Hmmmm let me see how I can improve this 👍